### DISCUSSION TITLE
Is there a way to monitor Coolify-hosted Next.js sites? Status pages or external tools?

### DISCUSSION THREAD

**Discussion by xer0xde** [coollabsio/coolify#4491] [Q&A]
Hi everyone,

I’ve been using Coolify to deploy and manage my Next.js applications, and I’m wondering if there’s any built-in feature (or recommended integration) for monitoring uptime/downtime. Specifically, I’m looking for:
	1.	A status page solution to display the current state of my deployments.
	2.	Any external tools that integrate well with Coolify for monitoring purposes.

Ideally, I’d prefer lightweight, easy-to-implement options that don’t require significant overhead. Open to both self-hosted and external SaaS recommendations.

Does anyone have experience with this or suggestions?

**Comment by ShaheerCH** [comment]
Hello
You can try Checkmate. It's open source and is present as a service on coolify
